### PR TITLE
OSDOCS-5146 added channel upgrade modules

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.8+

Issue:
https://issues.redhat.com/browse/OSDOCS-5146

Link to docs preview (VPN required):
Compliance:
[Updating the Compliance Operator](http://file.rdu.redhat.com/antaylor/OSDOCS-5146/security/compliance_operator/compliance-operator-updating.html)
[Compliance Operator release notes](http://file.rdu.redhat.com/antaylor/OSDOCS-5146/security/compliance_operator/compliance-operator-release-notes.html)

File Integrity:
[Updating the File Integrity Operator](http://file.rdu.redhat.com/antaylor/OSDOCS-5146/security/file_integrity_operator/file-integrity-operator-updating.html)
[File Integrity Operator release notes](http://file.rdu.redhat.com/antaylor/OSDOCS-5146/security/file_integrity_operator/file-integrity-operator-release-notes.html)

Additional information:
This PR brings in [OLM-based content](https://docs.openshift.com/container-platform/4.12/operators/admin/olm-upgrading-operators.html) regarding updating Operators, while removing duplicated content in the File Integrity Operator docs.

### CODE REVIEW

**Comment by sheriff-rh** [comment]
/label peer-review-needed

Engineering approval received in Slack.

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 1]
```diff
@@ -13,6 +13,8 @@ These release notes track the development of the Compliance Operator in the {pro
 
 For an overview of the Compliance Operator, see xref:../../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance-operator[Understanding the Compliance Operator].
 
+To access the latest release, see xref:../../security/compliance_operator/compliance-operator-updating.html#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
```
Comment (mjpytlak) [inline security/compliance_operator/compliance-operator-release-notes.adoc:16]:
```suggestion
To access the latest release, see xref:../../security/compliance_operator/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
```

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc [hunk 2]
```diff
@@ -13,14 +13,16 @@ These release notes track the development of the File Integrity Operator in the
 
 For an overview of the File Integrity Operator, see xref:../../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[Understanding the File Integrity Operator].
 
+To access the latest release, see xref:../../security/file_integrity_operator/file-integrity-operator-updating.html#olm-preparing-upgrade_file-integrity-operator-updating[Updating the File Integrity Operator].
```
Comment (mjpytlak) [inline security/file_integrity_operator/file-integrity-operator-release-notes.adoc:16]:
```suggestion
To access the latest release, see xref:../../security/file_integrity_operator/file-integrity-operator-updating.adoc#olm-preparing-upgrade_file-integrity-operator-updating[Updating the File Integrity Operator].
```

**Review by mjpytlak — COMMENTED** [review]
@sheriff-rh Just a few updates to your xrefs. Otherwise, LGTM. Thanks!

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc (hunk 2 above)
Comment (sheriff-rh) [reply security/file_integrity_operator/file-integrity-operator-release-notes.adoc:16]: Oh man, good catch. Thanks!

**Comment by sheriff-rh** [comment]
Previously received LGTM in Slack, moving to merge review. 

/label merge-review-needed

**Pull request merged**